db/migrate/20171011211344_create_installations.rb

```diff
@@ -0,0 +1,22 @@
+class CreateInstallations < ActiveRecord::Migration[5.1]
+  def change
+    create_table :installations do |t|
+      t.text :name, null: false
+      t.text :street_address
+      t.text :extended_address
+      t.text :locality
+      t.text :region
+      t.text :region_code
+      t.text :postal_code
+      t.text :country_name
+      t.text :country_code
+      t.float :latitude
+      t.float :longitude
+      t.json :email_addresses, default: []
```

Comment:
Is this a JSON array of KV, where key == e-mail address type and value is the address?

Reply:
The structure is more obvious now that the JSON seed files are prettified. But to your question, `email_addresses` is an array of objects that look like:

```json
{
  "email_address": "email@example.com",
  "note": "some text note/label"
}
```

Similarly, `phone_numbers` take the form:

```json
{
  "phone_number": "555555555",
  "phone_type": "voice",
  "dsn": false,
  "note": "some text note/label"
}
```

…and `urls` takes the form:

```json
{
  "url": "http://example.com",
  "note": "some text note/label"
}
```

I'd previously noted this structure [in a comment on PR #97](https://github.com/deptofdefense/move.mil/pull/97#discussion_r141938665).